torch/nn/utils/clip_grad.py

```diff
@@ -23,19 +21,22 @@
 ]
 
 
-_tensor_or_tensors = Union[
+_TensorOrTensors: TypeAlias = Union[
```

Comment:
ruff complained about the name of this once I exposed it as a TypeAlias, and it's an internal only TypeAlias